packages/react-native-web/src/modules/createDOMProps/index.js

```diff
@@ -148,7 +149,8 @@ const createDOMProps = (component, props, styleResolver) => {
     component === 'button' ||
     component === 'input' ||
     component === 'select' ||
-    component === 'textarea'
+    component === 'textarea' ||
+    component === 'label'
   ) {
     if (accessible === false || !focusable) {
```

Comment:
I'm not too sure what this should be for a label, but it seemed that it should have a `tabIndex=0`, so i left added htmlFor outside of here